.github/workflows/style.yml

```diff
@@ -10,20 +10,20 @@ on:
 jobs:
   style:
     name: Test code style
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
These changes also require a changelog entry and updating README.md.

Reply:
Thanks for the hint to README.md, I updated the information there. Since this is only about our test environment, I don't think we need a changelog entry (and I haven't found such an entry for earlier such updates).